test/nanoserver/nanoserver.tests.ps1

```diff
@@ -0,0 +1,15 @@
+Describe "Verify PowerShell Runs" {
```

Comment:
License header?

Reply:
@daxian-dbw  or @iSazonov  can you add this?

Reply:
Submitted #10171